islandora_bookmark.module

```diff
@@ -756,11 +762,12 @@ function islandora_bookmark_fedora_repository_object_form(array $form, array &$f
       $bookmark_object = islandora_bookmark_get_bookmark_by_number($value);
       $links[] = l($bookmark_object->bookmarkName, 'islandora-bookmark/listid/' . $bookmark_object->getId());
     }
-
+    // @see https://www.drupal.org/docs/7/api/localization-api/dynamic-or-static-links-and-html-in-translatable-strings.
     $form['islandora_bookmark']['lists'] = array(
       '#type' => 'item',
-      '#prefix' => '<h3>' . t('Bookmarked in') . ':</h3>',
-      '#markup' => theme('item_list', array('items' => $links)),
+      '#prefix' => t('<h3>Bookmarked in:</h3'),
```

Comment:
@jonathangreen https://github.com/Islandora/islandora_bookmark/pull/101/commits/7a598e4a8a6f6495a83a4e935b650861970c83c9 should fix the issue so translation stays stable. Only way to have coder pass.